src/interfaces/diff-package.transport.ts

```diff
@@ -8,14 +8,38 @@ export interface ConfigurationChangeTransport {
 export interface NodeDiffTransport {
     nodeKey: string;
     changes: ConfigurationChangeTransport[];
+    baseConfiguration: NodeConfiguration
```

Comment:
Can we end with semi-colon for all of these

Reply:
Changed it [here](https://github.com/celonis/content-cli/pull/189/commits/7ab2d7015ee7831f6563e372852778267701f271).